pkg/gui/controllers/files_controller.go

```diff
@@ -748,6 +755,90 @@ func (self *FilesController) createStashMenu() error {
 	})
 }
 
+func (self *FilesController) openCopyMenu() error {
+	copyNameNoItemSelectionErr := utils.ResolvePlaceholderString(
+		self.c.Tr.FileCopyNoItemSelectedError,
+		map[string]string{"target": "name"},
+	)
+	copyNameItem := &types.MenuItem{
+		Label: self.c.Tr.CopyFileName,
+		OnPress: func() error {
+			name := self.context().GetSelectedItemId()
```

Comment:
@stefanhaller I was fixing the code, and I realized that it's not that useful to copy an item name. What do you think about changing this code to copy its path instead?

Reply:
No strong opinion here, I have never missed either very much. But I could see both as potentially useful for some people, so why not offer both? (`n File name`, `p File path`). Just to be sure, we are talking about a relative path here, not an absolute one, right? So for top-level files both commands would yield the same result, but I don't see this as a problem.

Also, just mentioning that copying the path is already available with `<c-o>`, but I'm not sure that's an argument for or against anything. 😄

Reply:
To be honest, I wanted to suggest using the same functionality of the `<c-o>` shortcut, since copying the full path would also give you access to the file name. But I'm not sure if that makes sense for the users. Let's keep the original idea and postpone the copy-path feature. Would that work for you?

Reply:
If you only want one or the other, then I do think that path is the more useful of the two, so I'd pick that. However, I really don't know what's wrong with offering both; it's not much work, and as long as they don't compete for key bindings (which they don't), I don't see any downside.

Reply:
Yeah, you're right - I kept both of them.